.editorconfig

```diff
@@ -35,6 +35,11 @@ indent_size = 2
 end_of_line = lf
 indent_size = 2
 
+[Syntax.xml.Main.Generated.cs]
+# RS0027 does not work for types in a nullable context
```

Comment:
Why not? Do we have a bug tracking fixing this, or is this something that can't be fixed?

Reply:
No bug yet but I'll file one if this PR merges without a fix.

Reply:
I removed the suppression and it built locally, so trying to see if it passes the PR build as well.